### PR TITLE
the profile page, Issue #9

### LINKED ISSUE #9: Profile page for each fellow
When you click on each fellows card, it sends you to a separate page which includes more detail such as:

- Description
- Favourite programming languages
- Interests outside of technology
- University

### PR DESCRIPTION
**Pull request by 2leila5**
## Please, go through these steps before you submit a PR.

- [ ] My Pod Leader knows I'm working on this Pull Request
- [ ] I've explained what the Pull Request is adding.
- [ ] I've explained why this is important.

Fixes #9

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -9,6 +9,7 @@ GEM
       eventmachine (>= 0.12.9)
       http_parser.rb (~> 0.6.0)
     eventmachine (1.2.7)
+    eventmachine (1.2.7-x64-mingw32)
```
Comment (sumana2001) [inline Gemfile.lock:12]: Please don't commit the Gemfile.lock file

**Review by sumana2001 — CHANGES_REQUESTED** [review]
Screenshot 2023-03-11 at 8 45 02 PM
In some screen sizes, it is not responsive

https://deploy-preview-27--mlh-prep-23-mar-prep-1-portfolio.netlify.app/fellowpages/koichi
Koichi's page is not working for some reason

https://deploy-preview-27--mlh-prep-23-mar-prep-1-portfolio.netlify.app/fellowPages/
Sana's page is linking to this link which is not found

**Review by sumana2001 — CHANGES_REQUESTED** [review]
The Gemfile.lock file shouldn't be deleted, just removed from the commit. Also please fix merge conflicts

**Comment by J0SAL** [comment]
@sumana2001 can you please review the changes? Right now the Gemfile.lock is getting ignored cuz of .gitignore, is that expected behavior?

**Comment by J0SAL** [comment]
I reverted the Gemfile.lock to the version before it was modified, does it work?

**Pull request merged**